components/omega/src/ocn/TendencyTerms.cpp

```diff
@@ -26,12 +26,30 @@ int Tendencies::init() {
    int Err = 0;
 
    HorzMesh *DefHorzMesh = HorzMesh::getDefault();
-   Config *TendConfig;
+
+   // Get TendConfig group if available
+   Config *OmegaConfig = Config::getOmegaConfig();
+   Config TendConfig("Tendencies");
+   if (OmegaConfig->existsGroup("Tendencies")) {
+      Err = OmegaConfig->get(TendConfig);
+   }
 
    int NVertLevels = 60;
 
+   // Retrieve NVertLevels from Config if available
+   Config DimConfig("Dimension");
+   if (OmegaConfig->existsGroup("Dimension")) {
+      Err = OmegaConfig->get(DimConfig);
+      if (DimConfig.existsVar("NVertLevels")) {
+         Err = DimConfig.get("NVertLevels", NVertLevels);
+      }
+   }
```

Comment:
That could probably be done, curious what @philipwjones's opinion is since he wrote the Config class. Maybe just put a wrapper around the two functions. We might want to add else blocks that write to log "this var not found", but that could be done within a wrapper. I think it makes sense to keep the existsGroup and get group functions separate to block together fetching multiple variables from the same group.

Reply:
The Config get functions will return an error if they don't exist, but then you'd want to be checking the error code on return, so you'd be trading the pre-check on existence for a post-check on the error code returned. Similar number of lines, but maybe fewer indent levels on the post-check. So you'd have something like: `
Err = OmegaConfig->get(DimConfig)
if (Err != 0) {
  LOG_ERROR(message);
  return Err;
}
Err = DimConfig.get("NVertLevels",NVertLevels)
if (Err != 0) {
   // either log the error or set a default value
}
`

Reply:
Thanks @philipwjones. Personally, I think I prefer the post-check route. This also provides a consistent way to specify the default values, which would address some of my other comments as well.